srcpkgs/j/template

```diff
@@ -139,3 +139,14 @@ do_install() {
 	_jconsole_wrapper_script >bin/jc.sh
 	vbin "bin/jc.sh" jc
 }
+
+j-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/lib/pkgconfig
```

Comment:
None of these regexes will match, did you test this PR?
The only libraries in j are:
```
/usr/lib/j/j64/jconsole
/usr/lib/j/j64/libj.so
/usr/lib/j/j64avx/jconsole
/usr/lib/j/j64avx/libj.so
/usr/lib/j/j64avx2/jconsole
/usr/lib/j/j64avx2/libj.so
```
and they are required in the main package or jc won't work. What exactly do you want to split to -devel?